strawberryfields/ops.py

```diff
@@ -616,10 +617,28 @@ class Channel(Transformation):
     This class provides the base behaviour for non-unitary
     maps and transformations.
     """
-    # NOTE: At the moment this is an empty class, and only
-    # exists for a nicer inheritence diagram.
-    # This class will likely get filled in when we add more channels.
-    pass
+    def merge(self, other):
+        # check that other an identical channel, and that
+        # no extra dependencies <=> no RegRefTransform parameters
```

Comment:
"that there are no extra dependencies"?

Reply:
I assume so? This is one of Ville's original comments